src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,24 +14,34 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int orderNumber; //instance variable
+    private Birthday birthday;
 
     private final UniqueTagList tags;
+    private static int currentOrderNumber = 0; //class variable that can also be used to track total number of valid people in address book
```

Comment:
This is not the correct naming convention for a constant value (all upper cases with '_')